pyproject.toml

```diff
@@ -30,7 +30,9 @@ prometheus-client = "^0.21.0"
 openrelik-ai-common = "~0.5.0"
 openrelik-common = "^0.6.0"
 sse-starlette = "^2.3.6"
-
+opentelemetry-instrumentation-fastapi = "^0.53b1"
```

Comment:
Both opentelemetry libs are not needed if you update your openrelik-common lib!

Reply:
Great catch! This was brought in in a git conflict merge. I removed them